firebase-appdistribution-api/src/main/java/com/google/firebase/appdistribution/FirebaseAppDistribution.java

```diff
@@ -179,6 +182,76 @@ public interface FirebaseAppDistribution {
    */
   void startFeedback(@NonNull CharSequence infoText, @Nullable Uri screenshot);
 
+  /**
+   * Displays a notification that, when tapped, will take a screenshot of the current activity, then
+   * start a new activity to collect and submit feedback from the tester along with the screenshot.
+   *
+   * <p>On Android 13 and above, this method requires the <a
+   * href="https://developer.android.com/develop/ui/views/notifications/notification-permission">runtime
+   * permission for sending notifications</a>: {@code POST_NOTIFICATIONS}. If your app targets
+   * Android 13 (API level 33) or above, you should <a
+   * href="https://developer.android.com/training/permissions/requesting">request the
+   * permission</a>.
+   *
+   * <p>When the notification is tapped:
+   *
+   * <ol>
+   *   <li>If the app is open, take a screenshot of the current activity
+   *   <li>If tester is not signed in, presents the tester with a Google Sign-in UI
+   *   <li>Starts a full screen activity for the tester to compose and submit the feedback
+   * </ol>
+   *
+   * <p>On Android 8 and above, the notification will be created in its own notification channel.
+   *
+   * @param infoTextResourceId string resource ID of text to display to the tester before collecting
```

Comment:
Does the underlying api require a resource id? if not, do we want to restrict to only allow resource ids as opposed to allowing passing in custom strings like constants?


If resourceId is preferred, please make sure to annotate this param with `@StringRes`.

Reply:
My preference is in the other direction. If we eliminate one, I'd vote for the overload with `CharSequence` and only keep `@StringRes int`. The latter is a best practice. Hard-coded strings are smelly. Or we keep both, but add JavaDoc which explains that `@StringRes int` is preferred.

Reply:
That's a good point, Kai. I actually just noticed that if you use `TextView.setText(CharSequence)` you get a compiler warning: "String literal in setText can not be translated. Use Android resources instead."

So I'm tempted to remove the CharSequence option, but that would prevent the dev from using format strings: https://developer.android.com/guide/topics/resources/string-resource#formatting-strings

I feel like we should support that.

Reply:
Would it make sense to add an overload that would mirror `getResource(int, Object...)`? i.e. something like

```java
void showFeedbackNotification(
      @NonNull InterruptionLevel interruptionLevel, @StringRes int infoTextResourceId, Object... formatArgs);
```

Reply:
We could, but if they want to use a format string AND html elements like `<a>` tags they would have to do some extra steps and then final pass us a styled CharSequence: https://developer.android.com/guide/topics/resources/string-resource#StylingWithHTML

This might seem like an edge case, but because this text will likely include links to terms and conditions I expect anyone who wants to use a format string will have to do this.